src/main/java/backend/IssueMetadata.java

```diff
@@ -95,6 +102,10 @@ public boolean isUpdated() {
         return isUpdated;
     }
 
+    public String getIssueETag() {
```

Comment:
I feel it is, unless there can be more ETags in metadata in future. We'll need the metadata instance to call it, so it shouldn't be too confusing.
```java
issue.getMetadata().getETag()
model.getUpdateSignature().issuesETag
```